components/dashboard/src/settings/Preferences.tsx

```diff
@@ -129,7 +128,7 @@ export default function Preferences() {
                             </ul></InfoBox>
                         }
                         <p className="text-left w-full text-gray-500">
-                            The <strong>JetBrains desktop IDEs</strong> are currently in beta. <a href="https://github.com/gitpod-io/gitpod/issues/6576" target="gitpod-feedback-issue" rel="noopener" className="gp-link">Send feedback</a> · <a href="https://www.gitpod.io/docs/integrations/jetbrains" target="_blank" rel="noopener" className="gp-link">Documentation</a>
+                            The <strong>JetBrains desktop IDEs</strong> are currently in beta. <a href="https://github.com/gitpod-io/gitpod/issues/6576" target="gitpod-feedback-issue" rel="noopener" className="gp-link">Send feedback</a> · <a href="https://www.gitpod.io/docs/integrations/jetbrains" target="_blank" rel="noopener noreferrer" className="gp-link">Documentation</a>
```

Comment:
let's tackle dashboard refacoting later, first we need to make sure that something is running on `ide.gitpod.io` and we can fetch images from it

Reply:
> let's tackle dashboard refacoting later

:100: also avoids rollout trouble